loop/management/commands/helpline_queue.py

```diff
@@ -40,6 +40,7 @@ def check_pending_or_not(self,incoming_call_id):
             return ''
 
     def handle(self, *args, **options):
+        # Check if State filter is required or not
```

Comment:
yes we need expert state filter here because we need to make a call to the user here.